test/command/cmd.go

```diff
@@ -210,11 +210,12 @@ func (s *Starter) KillAll() {
 }
 
 // Clean cleans all temporary directories and processes.
-func (s *Starter) Clean() {
+func (s *Starter) Clean() error {
```

Comment:
Do you still need the return value `error`? I see here only return nil

Reply:
@yeya24 
You are right,I changed it.